data/runtime/example-config.R

```diff
@@ -0,0 +1,7 @@
+DATAVERSE_KEY <- "" # API Access Key
+DATAVERSE_SERVER <- "https://demo.dataverse.org"  # for production: "dataverse.harvard.edu" # Which dataverse server to use
+DATAVERSE_VERSEID <- "covid-rd" # top level dataverse within which data sets should be published
```

Comment:
rd? Is this final? I am not sure I get it.

Reply:
https://dataverse.harvard.edu/dataverse/covid-rt 
RT, just me being a wally.